forgettable-frontend/src/pages/SignInPage/SignInPage.module.css

```diff
@@ -36,9 +38,9 @@
     height: 60px;
     font-size: 20px;
```

Comment:
This font size is hard-coded, do we want to use an existing font size from global CSS?

Reply:
+1